<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JuniorAdminWindow</class>
 <widget class="QMainWindow" name="JuniorAdminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1131</width>
    <height>654</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MedScheduler</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res.qrc">
    <normaloff>:/images/icon-512.png</normaloff>:/images/icon-512.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QGroupBox" name="menu">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>20</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;
background-color: rgb(31, 31, 39);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="title">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>20</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 20pt &quot;Segoe UI&quot;;</string>
         </property>
         <property name="text">
          <string>MedScheduler</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_get_users">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>10</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
           color: rgb(255, 255, 255);
         font: 15pt 'Arial';
         text-align: left;
          padding-left: 10%;
        background-color: transparent;
        border: none;
        }
        QPushButton:hover {
          background-color: rgba(64, 64, 80, 100);
          border-radius: 10px;
        }</string>
         </property>
         <property name="text">
          <string>Get users table</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_get_doctors">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>10</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
           color: rgb(255, 255, 255);
         font: 15pt 'Arial';
         text-align: left;
          padding-left: 10%;
        background-color: transparent;
        border: none;
        }
        QPushButton:hover {
          background-color: rgba(64, 64, 80, 100);
          border-radius: 10px;
        }</string>
         </property>
         <property name="text">
          <string>Get doctors table</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_add_doctor">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>10</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
           color: rgb(255, 255, 255);
         font: 15pt 'Arial';
         text-align: left;
          padding-left: 10%;
        background-color: transparent;
        border: none;
        }
        QPushButton:hover {
          background-color: rgba(64, 64, 80, 100);
          border-radius: 10px;
        }</string>
         </property>
         <property name="text">
          <string>Add doctor</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_attach_doctor">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>10</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
           color: rgb(255, 255, 255);
         font: 15pt 'Arial';
         text-align: left;
          padding-left: 10%;
        background-color: transparent;
        border: none;
        }
        QPushButton:hover {
          background-color: rgba(64, 64, 80, 100);
          border-radius: 10px;
        }</string>
         </property>
         <property name="text">
          <string>Attach/detach doctor</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_add_appointment">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>10</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
           color: rgb(255, 255, 255);
         font: 15pt 'Arial';
         text-align: left;
          padding-left: 10%;
        background-color: transparent;
        border: none;
        }
        QPushButton:hover {
          background-color: rgba(64, 64, 80, 100);
          border-radius: 10px;
        }</string>
         </property>
         <property name="text">
          <string>Add an appointent</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_get_schedule">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>10</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
           color: rgb(255, 255, 255);
         font: 15pt 'Arial';
         text-align: left;
          padding-left: 10%;
        background-color: transparent;
        border: none;
        }
        QPushButton:hover {
          background-color: rgba(64, 64, 80, 100);
          border-radius: 10px;
        }</string>
         </property>
         <property name="text">
          <string>Get schedule</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>40</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>80</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="page_get_users" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="get_users_table_button">
          <property name="text">
           <string>Get users table</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="users_scroll">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="users_scroll_widget">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>842</width>
             <height>594</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_get_doctors" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="get_doctors_table_button">
          <property name="text">
           <string>Get doctors table</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="doctors_scroll">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="doctor_scroll_widget">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>842</width>
             <height>594</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_add_doctor" native="true">
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="last_name_label">
          <property name="text">
           <string>Last name</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="last_name_form"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="first_name_label">
          <property name="text">
           <string>First name</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="first_name_form"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="middle_name_label">
          <property name="text">
           <string>Middle name</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="middle_name_form"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="phone_number_label">
          <property name="text">
           <string>Phone number</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="phone_number_form"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="education_label">
          <property name="text">
           <string>Education</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="education_form"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="specialt_label">
          <property name="text">
           <string>Specialty</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="specialty_form"/>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="experience_label">
          <property name="text">
           <string>Experience</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="experience_form"/>
        </item>
        <item row="8" column="0">
         <widget class="QPushButton" name="add_doctor_button">
          <property name="text">
           <string>Add doctor</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QLabel" name="add_doctor_error">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 0, 0);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLineEdit" name="price_form"/>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="price_label">
          <property name="text">
           <string>Price</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_attach_doctor" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>35</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Data enter</string>
          </property>
          <layout class="QFormLayout" name="formLayout_2">
           <item row="2" column="1">
            <widget class="QLineEdit" name="hospital_id_form">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="doctor_id_form">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="doctor_id_label">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Doctor id</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="hospital_id_label_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Hospital id</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="error_label_add_remove">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>80</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 0, 0);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="add_doctor_to_hospital_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>30</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Add doctor</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="remove_doctor_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>30</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Remove doctor</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="empty_label">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_add_appointment" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Data enter</string>
          </property>
          <layout class="QFormLayout" name="formLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="doctor_id_label_2">
             <property name="text">
              <string>Doctor id</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="doctor_id_form_from_appointment"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="date_label">
             <property name="text">
              <string>Date</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="date_form"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="time_label">
             <property name="text">
              <string>Time</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="time_form"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="hospital_id_label">
             <property name="text">
              <string>Hospital id</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="hospital_id_form_from_appointment"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="cabinet_label">
             <property name="text">
              <string>Cabinet</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="cabinet_form"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="error_appointment">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 0, 0);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="add_appointment_button">
          <property name="text">
           <string>Add an appointment</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_get_schedule" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Enter doctor id</string>
          </property>
          <layout class="QFormLayout" name="formLayout_4">
           <item row="0" column="0">
            <widget class="QLabel" name="doctor_label">
             <property name="text">
              <string>Doctor id</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="doctor_id_schedule_form"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="error_schedule">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 0, 0);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="get_schedule_button">
          <property name="text">
           <string>Get schedule</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="schedule_scroll">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="schedule_widget">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>842</width>
             <height>481</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
